server/endpoint/endpoint.go

```diff
@@ -15,16 +14,19 @@ type Config struct {
 	Service *service.Service
 }
 
+// Endpoint is the endpoint collection.
+type Endpoint struct {
+	Healthz *healthz.Endpoint
+	Version *versionendpoint.Endpoint
+}
+
 func New(config Config) (*Endpoint, error) {
 	var err error
 
 	var healthzEndpoint *healthz.Endpoint
 	{
 		c := healthz.Config{
 			Logger: config.Logger,
-			Services: []healthzservice.Service{
-				config.Service.Healthz,
-			},
```

Comment:
Removed the health check. We use the default one for simple service availability.